app/code/community/Hackathon/EmailPreview/etc/config.xml

```diff
@@ -181,6 +387,22 @@
                         <class>hackathon_emailpreview/mail_type_rmaNew</class>
                         <method>hackathonEmailpreviewRenderEmailBefore</method>
                     </hackathon_emailpreview_rma_email>
+                    <hackathon_emailpreview_contact>
+                        <class>hackathon_emailpreview/mail_type_contact</class>
+                        <method>hackathonEmailpreviewRenderEmailBefore</method>
+                    </hackathon_emailpreview_contact>
+                    <hackathon_emailpreview_admin_password_forgot>
+                        <class>hackathon_emailpreview/mail_type_adminPasswordForgot</class>
```

Comment:
This class is missing in the PR, causing a warning (and therefore error in developer mode). Please can you add it?

Reply:
I'll try to check it up these days, maybe next friday since I have a day off...